src/pages/index.tsx

```diff
@@ -13,7 +13,7 @@ const Home = () => {
     useOpenLiveChat()
 
     return (
-        <Layout is_nav_transparent margin_top="0">
+        <Layout is_nav_transparent type="sticky" margin_top="0">
```

Comment:
Does `is_nav_transparent` and `type="sticky"` doing the same thing?

Reply:
thank u @sean-binary I didn't see this before. cause I just updated my branch with master. I'll fix that